app/src/main/kotlin/com/example/android/picover/presentation/ProfileDrawerContent.kt

```diff
@@ -0,0 +1,27 @@
+package com.example.android.picover.presentation
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.ModalDrawerSheet
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import com.intive.picover.R
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun ProfileDrawerContent() {
+	ModalDrawerSheet {
+		Box(
+			modifier = Modifier.fillMaxSize(),
+		) {
+			Text(
+				text = stringResource(id = R.string.Profile),
```

Comment:
Minor: `id` named parameter could be dropped in that case as for `stringResource` it is obvious that we are passing string id 😄

Reply:
naturlich :D changed :)